spec/latest/core/core.jsonld

```diff
@@ -442,19 +443,28 @@
       "@type": "rdf:Property",
       "label": "variable representation",
       "comment": "The representation format to use when expanding the IRI template.",
+      "range": "hydra:VariableRepresentation",
       "vs:term_status": "testing"
     },
     {
-      "@id": "hydra:BasicRepresentation",
+      "@id": "hydra:VariableRepresentation",
       "@type": "hydra:Class",
       "subClassOf": "hydra:Resource",
+      "label": "representation of variable values",
+      "comment": "A representation specifies how to serialize variable values into strings.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:BasicRepresentation",
+      "@type": "hydra:VariableRepresentation",
+      "subClassOf": "hydra:Resource",

```

Comment:
We is this a sub class of `hydra:Resource` and not an instance thereof? Same below